android/app/src/main/java/com/dismoi/scout/accessibility/BackgroundModule.kt

```diff
@@ -91,25 +92,29 @@ class BackgroundModule(@Nonnull reactContext: ReactApplicationContext) :
     promise.resolve("")
   }
 
-  @ReactMethod
-  fun testWithXpath(htmlString: String, xPathParser: String, promise: Promise) {
-    // create an instance of HtmlCleaner
-    val tagNode = HtmlCleaner().clean(
-      htmlString
-    )
-    val doc = DomSerializer(
-      CleanerProperties()
-    ).createDOM(tagNode)
-
-    val xpath: XPath = XPathFactory.newInstance().newXPath()
-
-    val str = xpath.evaluate(
-      xPathParser,
-      doc, XPathConstants.STRING
-    ) as String
-
-    promise.resolve(str)
-  }
+  // @ReactMethod
+  // fun testWithXpath(htmlString: String, xPathParser: String, promise: Promise) {
```

Comment:
We’ve removed XPath functionality ?

Reply:
Yes we did. Discussion here: https://github.com/dis-moi/mobile_poc/issues/200

Reply:
I did not understand this as a decision of removal, rather than a warning about the implications :

> Comme discuté il y aura deux options ici :
> 
>  - Résoudre le problème dans le code
>  - Suspendre cette feature
> 
> Je vous donne les implications de l'option 2 afin que vous puissiez la mettre en balance avec la première :

Nevertheless I’m going to see if I can get to the bottom of #200 …